search-service/src/main/kotlin/com/egm/stellio/search/web/EntityOperationHandler.kt

```diff
@@ -401,4 +424,17 @@ class EntityOperationHandler(
             )
         }
     }
+
+    private suspend fun getNgsiLdEntities(
```

Comment:
you could move it near `expandAndPrepareBatchOfEntities` and `checkBatchRequestBody` to co-locate code that is in the same "process"

Reply:
tried to find a more meaningful name for this function but no great idea so far...

Reply:
maybe :
 - prepareEntities() 
 - prepareBatchEntity()
 - prepareEntitiesFromRequestBody()

Reply:
`prepareEntitiesFromRequestBody` LGTM